squarelet/templates/organizations/organization_list.html

```diff
@@ -1,50 +1,97 @@
 {% extends "base.html" %}
-{% load static i18n avatar %}
+{% load static i18n avatar django_vite %}
+
 {% block title %}Organizations{% endblock %}
 
+{% block vite %}
+{{ block.super }}
+{% vite_asset 'frontend/views/organization_list.ts' %}
+{% endblock vite %}
+
 {% block content %}
-  <div class="_cls-content">
-    <h2>Organizations</h2>
-    <div class="_cls-action">
-      <a class="_cls-nostyle" href="{% url "organizations:create" %}">
-        Create a new organization
+<article id="organization_list">
+  <header>
+    {% if user.is_authenticated %}
+    <nav class="return">
+      <a href="{% url "users:detail" username=user.username %}">
+        {% include "core/icons/arrow-left.svg" %}
+        {% trans "Back to your account" %}
       </a>
-    </div>
-  </div>
-  
-  <div class="_cls-expandedContent">
-    <div id="_id-autocomplete" class="_cls-autocomplete">
-      <div class="_cls-inputSection">
-        <div class="_cls-input">
-          <div class="_cls-searchCircle"></div>
-          <div class="_cls-searchRectangle"></div>
-          <div id="_id-close" class="_cls-closeButton">
-            <div class="_cls-closeOpaque"></div>
-            <div class="_cls-closeLeft"></div>
-            <div class="_cls-closeRight"></div>
+    </nav>
+    {% endif %}
+    <h1>{% trans "Join an organization" %}</h1>
+  </header>
+
+  <div class="content">
+    <section class="search">
+      <h2>{% trans "Find an organization to join" %}</h2>
+      <div id="org_search"></div>
+    </section>
+
+    <section class="invites">
+      <h2>
+        {% trans "Pending invitations" %}
+      </h2>
+      {% if not has_pending %}
+      <div class="empty">
+        <p>{% trans "You don't have any outstanding invitations." %}</p>
+        <p>{% trans "Any invitations you receive will appear here." %}</p>
+      </div>
+      {% endif %}
+
```

Comment:
While looking at the code for user.get_pending_invitations(), it looks like we return none when the user doesn't have a confirmed email. A check for that here, and a link to send an email confirmation message, would be a great enhancement here.